ci/ci/constants.py

```diff
@@ -1,32 +1,34 @@
-from typing import Optional
+from typing import Optional, List
 
 GITHUB_CLONE_URL = 'https://github.com/'
 GITHUB_STATUS_CONTEXT = 'ci-test'
+TEAMS = ['Services', 'Compiler']
 
 
 class User:
-    def __init__(self, gh_username: str, hail_username: Optional[str]):
+    def __init__(self, gh_username: str, hail_username: Optional[str] = None, teams: List[str] = []):
```

Comment:
In situations like this, we tend to stick a `*` before `hail_username` to force the subsequent arguments to be keyword-only. I suppose in this case you'll never have a teams and not a hail_username, so this is probably the right choice actually.